.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,40 @@
+<!--
+    We do not guarantee a quick answer on GitHub.
+    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask)
+    to receive an answer shortly.
+-->
+
+### I'm using ...
+<!--
+    Checked checkbox should look like this: [x]
+-->
+- [ ] React Grid
+- [ ] React Chart
+- [ ] React Scheduler
+
+### Description
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]-->
+
+
+
+### Preferred Solution
+<!-- A clear and concise description of what you want to happen. -->
+
+
+
+### Additional context
+<!-- Add any other context or screenshots about the feature request here. -->
+
+
+
+### Environment
+<!---
+    Include as many relevant details about the environment with which you experienced the bug. Leave "none" if you don't have the package in your package.json file.
+-->
+
+- devextreme-reactive: xxx
+- react: xxx
+- browser: xxx
+- bootstrap: none
+- react-bootstrap: none
+- material-ui: none
```

Comment:
Should we add this section in the feature request?
Also, I think we can mention that customer can create own PR with a feature.

Reply:
I think this section can be removed.
I'll add note about PR